force-app/main/default/classes/MessageCtrl.cls

```diff
@@ -1,33 +1,133 @@
+global class MessageCtrl {
 
-        String authorizationId = 'ACb72ab7ad5500917d2d058ec77eaad682';
-        String authorizationToken= '1e3ec4de1589d8d6d09b213401dc8f0f';
-        String endPoint = 'https://api.twilio.com';
-        String sender = '+13133273959';
-        String reciever = '+918950025769';
-        String messageBody = 'Hello';
+    public static String getAuthorizationId() {
+
+        String accountSId = null;
+            // Create HTTP instance
+        Http http = new Http();
+            
+            // Create HTTP request
+            HttpRequest request = new HttpRequest();
+            request.setEndpoint('callout:Twilio/2010-04-01/Accounts');
+            request.setMethod('GET');
+        
+            // Send the request and get the response
+            HttpResponse response = http.send(request);
+        
+            // Check the response status
+            if (response.getStatusCode() == 200) {
+                // Parse the response body
+                // 
+                Dom.Document document = new Dom.Document();
+                document.load(response.getBody());
+                DOM.XMLNode root = document.getRootElement().getChildElement('Accounts', null).getChildElement('Account',null);
+                accountSId = root.getChildElement('OwnerAccountSid', null).getText();                
+            }
+        return accountSId;
+    }
+
+    @AuraEnabled()
+    public static String sendMessage(String senderNumber, String receiverNumber, String messageBody){
+
+        String authorizationId = getAuthorizationId();
 
         Http http = new Http();
 
         //Create Http Request
         HttpRequest request = new HttpRequest();
-        request.setEndpoint(endPoint+'/2010-04-01/Accounts/'+authorizationId+'/Messages.json');
+        request.setEndpoint('callout:Twilio/2010-04-01/Accounts/'+authorizationId+'/Messages.json');
         request.setMethod('POST');
-
-        //Create Authorization Header
-        Blob headerValue = Blob.valueOf(authorizationId+':'+authorizationToken);
-        String authorizationHeader = 'BASIC '+EncodingUtil.base64Encode(headerValue);
-        request.setHeader('Authorization', authorizationHeader);
+        request.setHeader('Content-Type', 'application/x-www-form-urlencoded');
 
         //Create request body
-        request.setBody('To=' + EncodingUtil.urlEncode(reciever, 'UTF-8') + '&From='+ EncodingUtil.urlEncode(sender,'UTF-8') + '&Body=' + EncodingUtil.urlEncode(messageBody,'UTF-8'));
+        request.setBody('To=' + EncodingUtil.urlEncode(receiverNumber, 'UTF-8') + '&From='+ EncodingUtil.urlEncode(senderNumber,'UTF-8') + '&Body=' + EncodingUtil.urlEncode(messageBody,'UTF-8'));
 
         //Send the request and get response
         HttpResponse response = http.send(request);
 
-        return response;
+        if(response.getStatusCode()==Integer.valueOf('201')){
```

Comment:
Any specific reason to typeCast the status code to Integer as I think it is Integer by default it self